src/Compilers/CSharp/Portable/Symbols/Source/SourceFieldLikeEventSymbol.cs

```diff
@@ -239,6 +253,96 @@ internal SourceEventDefinitionAccessorSymbol(
             {
                 return null;
             }
+
+            protected override SourceMemberMethodSymbol? BoundAttributesSource
+            {
+                get
+                {
+                    return IsExtern && this.MethodKind == MethodKind.EventAdd
+                        ? (SourceMemberMethodSymbol?)this.AssociatedEvent.RemoveMethod
```

Comment:
If the event is `extern`, are we taking the attributes from the `remove` accessor? If so, is this spec'ed?

Reply:
Yes, I don't know if it's spec'ed, but it's a pre-existing behavior:
 https://github.com/dotnet/roslyn/blob/80ccf3fe1914f0df2b3745e29fadfa7f87dd6431/src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedEventAccessorSymbol.cs#L49-L57

and I'm just replicating this for `partial` events. This issue is probably related: https://github.com/dotnet/roslyn/issues/76882